packages/tpetra/core/src/Tpetra_Details_Behavior.cpp

```diff
@@ -15,6 +15,7 @@ namespace Details {
 
 namespace BehaviorDetails {
 std::map<std::string, std::map<std::string, bool> > namedVariableMap_;
+bool verboseDisabled_ = false;
```

Comment:
Did you mean to include these changes here?  They look like they belong to some earlier PR.

Reply:
No, this was the change I put in to test the StaticProfile test (where I wanted verbose output for only a part of the test).

Reply:
To be clear.  Yes, I meant to include these changes.  It was in this work that I wrote this stuff.